src/components/TaskHeader.js

```diff
@@ -0,0 +1,120 @@
+import Text from './Text';
+import participantPropTypes from './participantPropTypes';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import Icon from './Icon';
+import MenuItemWithTopDescription from './MenuItemWithTopDescription';
+import Button from './Button';
+import * as TaskUtils from '../libs/actions/Task';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: reportPropTypes.isRequired,
+
+    /** Personal details so we can get the ones for the report participants */
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+function TaskHeader(props) {
+    const title = ReportUtils.getReportName(props.report);
+    const assigneeName = ReportUtils.getDisplayNameForParticipant(props.report.managerEmail);
+    const assigneeAvatar = ReportUtils.getAvatar(lodashGet(props.personalDetails, [props.report.managerEmail, 'avatar']), props.report.managerEmail);
+    const isOpen = props.report.stateNum === CONST.REPORT.STATE_NUM.OPEN && props.report.statusNum === CONST.REPORT.STATUS.OPEN;
+    const isCompleted = props.report.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.report.statusNum === CONST.REPORT.STATUS.APPROVED;
+    const parentReportID = props.report.parentReportID;
+
+    useEffect(() => {
+        TaskUtils.setTaskReport(props.report);
+    }, [props.report]);
+
+    return (
+        <View style={styles.borderBottom}>
+            <View style={[{backgroundColor: themeColors.highlightBG}, styles.pl0]}>
+                <View style={[styles.ph5, styles.pb5]}>
+                    <Text style={[styles.textLabelSupporting, styles.lh16]}>{props.translate('common.to')}</Text>
+                    <TouchableOpacity
+                        onPress={() => Navigation.navigate(ROUTES.getTaskReportAssigneeRoute(props.report.reportID))}
+                        disabled={!isOpen}
+                    >
+                        <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, styles.pv3]}>
+                            <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                                {props.report.managerEmail && (
+                                    <>
+                                        <Avatar
+                                            source={assigneeAvatar}
+                                            type={CONST.ICON_TYPE_AVATAR}
+                                            name={assigneeName}
+                                            size={CONST.AVATAR_SIZE.HEADER}
+                                        />
```

Comment:
This caused a regression. Previously we used `TaskSelectorLink` -> `MultipleAvatars`. And `MultipleAvatars` wraps the `Avatar` with a `Tooltip` https://github.com/Expensify/App/blob/5743b159f7ae44fb792f87d860b161c6b5c7bbf1/src/components/MultipleAvatars.js#L69 Now we are missing the tooltip (Coming from https://github.com/Expensify/App/issues/21713).